lib/oath/keys.dart

```diff
@@ -18,7 +18,7 @@ import 'package:flutter/material.dart';
 
 const _prefix = 'oath.keys';
 
-const setOrManagePasswordAction = Key('$_prefix.set_or_manage_oath_password');
+const setOrManagePasswordAction = Key('$_prefix.set_or_manage_w_password');
```

Comment:
is this change intended? should be better with `set_or_manage_password`. note that this is not a localization key but a string for Widget key.